test/test_allocation_domain.cpp

```diff
@@ -1053,4 +1053,34 @@ TEST_F(AllocationDomainTest, NHWC2d_To_NHWC2d_cacheFork_CUDA) {
   testValidate(&fusion, cg_outputs, {t0}, {t0, t0}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, AllocationDomainVectorizationIssue902) {
```

Comment:
changes on codegen switched this fusion from pointwise scheduler to transpose scheduler. Which fixed the vectorization on non-inner most dimension issue.

Unfortunately we are not getting vectorization on both read / write. I was aware of that issue already, this is a note that I need to patch that as well.